CoordinatorProject/CoordinatorProject/Coordinators/SecondCoordinator.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  SecondCoordinator.swift
+//  CoordinatorProject
+//
+//  Created by Daniil on 28.12.2022.
+//
+
+import UIKit
+
+/// Координатор второго контроллера
```

Comment:
```suggestion
/// Координатор второй
```
Он не принадлежит 1 му контролеру как роутер